tasks/main.yml

```diff
@@ -43,6 +43,7 @@
 
 - name: Reboot Riak to accept the new config and ensure it is enabled to start at bootup
   service: name=riak enabled=yes state=restarted
+  when: riak_init_system == 'system'
```

Comment:
OK, the default is called `system`. It just seemed close enough to `systemd` that I thought I'd ask.

PR seems fine to me.